### PR TITLE
[wasm] Ensure prerequisites are installed on the perf pipeline

### PR DESCRIPTION
**Pull request by radical**

### CODE REVIEW

File: eng/pipelines/coreclr/templates/run-performance-job.yml [hunk 1]
```diff
@@ -58,9 +58,35 @@ jobs:
...
+        pip3 install azure.storage.queue==12.0.0 &&
+        sudo apt-get update &&
+        sudo apt -y install curl dirmngr apt-transport-https lsb-release ca-certificates &&
+        curl -fsSL https://deb.nodesource.com/setup_18.x | sudo -E bash - &&
+        sudo apt-get -y install nodejs &&
+        npm install --prefix $HELIX_WORKITEM_PAYLOAD jsvu -g &&
+        $HELIX_WORKITEM_PAYLOAD/bin/jsvu --os=linux64 --engines=v8,javascriptcore &&
+        export PERFLAB_UPLOAD_TOKEN="$(HelixPerfUploadTokenValue)" &&
+        export PERF_PREREQS_INSTALLED=1;
+        test "x$PERF_PREREQS_INSTALLED" = "x1" || echo "** Error: Failed to install prerequites **"
```
Comment (adamsitnik) [inline eng/pipelines/coreclr/templates/run-performance-job.yml:88]: we should consider moving this logic to an `.sh` file

**Review by adamsitnik — APPROVED** [review]
LGTM, thank you @radical !

**Comment by LoopedBard3** [comment]
@radical it appears that this is creating errors on windows micro builds: https://dev.azure.com/dnceng/internal/_build/results?buildId=1929747&view=results. Specifically: '"x$PERF_PREREQS_INSTALLED" was unexpected at this time.'. I think this is due to windows builds not supporting the bash-style if statements (could be wrong on this). I noticed that the prereqs envvar was not added to the windows precommands, would duplicating the command without the if statement and conditioning based on if we are on Windows_NT (like Condition="'$(AGENT_OS)' != 'Windows_NT'") work with these changes as expected?

**Comment by radical** [comment]
Yes, adding `Condition="'$(AGENT_OS)' != 'Windows_NT'"` so the command is added only for !windows, would work.

**Comment by radical** [comment]
> Yes, adding `Condition="'$(AGENT_OS)' != 'Windows_NT'"` so the command is added only for !windows, would work.

Are you opening a PR, or should I?

**Comment by LoopedBard3** [comment]
I can open a PR. 👍

**Pull request merged**